lib/apipie/extractor/recorder.rb

```diff
@@ -150,8 +150,8 @@ def analyze(env, &block)
       end
 
       module FunctionalTestRecording
-        def process(*args) # action, parameters = nil, session = nil, flash = nil, http_method = 'GET')
-          ret = super(*args)
+        def process(*, **) # action, parameters = nil, session = nil, flash = nil, http_method = 'GET')
```

Comment:
Isn't it better to use

```ruby
def process(*args, **kargs)
  ret = super
  # ...
end
```
?

Reply:
You can. I see both in the rails core.

Reply:
If it would make you accept the PR, I'll change it ...

Reply:
Nevermind, seems OK.